sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorClient/MetricFeedbackLiveTests.cs

```diff
@@ -0,0 +1,290 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.AI.MetricsAdvisor.Models;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.MetricsAdvisor.Tests
+{
+    public class MetricFeedbackLiveTests : MetricsAdvisorLiveTestBase
+    {
+        public MetricFeedbackLiveTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        private DateTimeOffset CreatedFeedbackStartTime => DateTimeOffset.Parse("2020-09-26T00:00:00Z");
+
+        private DateTimeOffset CreatedFeedbackEndTime => DateTimeOffset.Parse("2020-09-29T00:00:00Z");
```

Comment:
applies to the other 2 date properties

Reply:
Not these two. These are only used to specify the time range of data to which the feedback applies, but it won't prevent the creation of the feedback. We're using these to test `Add` (followed by `Get`) methods.

Changing the other two (`FeedbackSamplingStartTime` and `FeedbackSamplingEndTime`) could make the tests fail. We're using these in the `GetAll` test. If they're changed to a time range in which, by chance, no feedback exists, we'll throw because I added the following check to the end of the method:
```cs
Assert.That(feedbackCount, Is.GreaterThan(0));
```
to make sure we're actually testing something.

Reply:
could you add a comment for those 2 properties (`FeedbackSamplingStartTime` and `FeedbackSamplingEndTime`) so if someone else ever wants to update the tests, they know they can't update this values?
Other option is to rename them and include `GetAll` or something similar to imply the relationship